src/DigitalTwin/pnp/src/common/utility.ts

```diff
@@ -38,12 +38,10 @@ export class Utility {
    * replace all for content
    * @param str string
    * @param replacement replacement
-   * @param caseInsensitive identify if it is case insensitive
    */
-  static replaceAll(str: string, replacement: Map<string, string>, caseInsensitive = false): string {
-    const flag = caseInsensitive ? "ig" : "g";
+  static replaceAll(str: string, replacement: Map<string, string>): string {
     const keys = Array.from(replacement.keys());
-    const pattern = new RegExp(keys.join("|"), flag);
+    const pattern = new RegExp(keys.join("|"), "g");
```

Comment:
If this module is a general utility I would not suggest to cut it's ability to handle case insensitive condition. Just in case in the future people may want such as a function to do something.

Reply:
This function is to replace placeholder for template file. Since the placeholder is defined by ourselves and there is really no need to support case insensitive for placeholder in the future, I remove this logic.

In addition, the current logic doesn't work for case insensitive. Although regular expression can find matched word (digitalTwinIdentifier) with case insensitive, for the replacement, we need to make the keys (DigitalTwinIdentifier) in the replacement map as lower case at first, so that we could find the correct key. The pre-process complicates the code and in most case there is no need for placeholder case insensitive.